src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html * {
+  border: none;
+  outline: none;
+  background-color: #fff0;
+  font-family: Avenir, sans-serif;
+}
+
+.searchContainer {
+  box-sizing: border-box;
```

Comment:
move `box-sizing` properly to `*` selector

Reply:
like this ?
* { box-sizing: border-box; }

Reply:
moved box-sizing to  html * {}